torch_xla/runtime.py

```diff
@@ -220,8 +220,17 @@ def addressable_runtime_device_count() -> int:
   return torch_xla._XLAC._xla_num_runtime_devices()
 
 
+def disable_spmd():
+  """API to dsiable SPMD mode.
+  Revert the environment variable set by `use_spmd`, reset  
+  """
+  os.environ["XLA_USE_SPMD"] = "0"
+  torch_xla._XLAC._set_spmd_mode(False)
+
+
 # TODO(yeounoh) introduce SPMD configuration.
-def use_spmd(auto: Optional[bool] = False):
+def use_spmd(auto: Optional[bool] = False,
+             force_tensors_on_spmd_device: Optional[bool] = False):
```

Comment:
nit: the word "force" is meaningless here IMO. I think a more descriptive name could be `replicate_existing_tensors`.